src/helpers/adapters/mysql.ts

```diff
@@ -155,7 +155,20 @@ export async function storeVote(space, body, ipfs, receipt, id) {
     vp_state: ''
   };
 
-  await db.queryAsync('INSERT IGNORE INTO votes SET ?', params);
+  const connection = await db.getConnectionAsync();
+
+  await connection.beginTransactionAsync();
```

Comment:
Hum I think transaction is not ideal cuz between start and commit the table canot be updated by others queries